src/test/java/fr/uge/lp/databackbonepoc/services/AlgorithmServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package fr.uge.lp.databackbonepoc.services;
+
+import fr.uge.lp.databackbonepoc.algorithms.Algorithm;
+import fr.uge.lp.databackbonepoc.configurations.AlgorithmConfig;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
And remove import .*

```suggestion
import fr.uge.lp.databackbonepoc.algorithms.Algorithm;
import fr.uge.lp.databackbonepoc.configurations.AlgorithmConfig;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import java.util.List;
import static org.junit.jupiter.api.Assertions.*;
```